templates/result-email-template.html

```diff
@@ -72,13 +98,23 @@ <h2>General Information</h2>
             <td>{{ log_link }}</td>
         </tr>
         <tr>
-            <th style="color: blue">Total suite duration</th>
-            <td>{{ suite_run_time }}</td>
+            <th style="color: blue">Invoked by User</th>
+            <td>{{ trigger_user }}</td>
+        </tr>
+    </table>
+    <h3>Suite duration</h3>
+    <table class="half-table">
+        <tr>
+            <th style="color: blue">Start Time</th>
+            <td>{{ suite_run_time.start }}</td>
```

Comment:
can we have the name representation here instead of number format. This may confuse the geographical lookers :).  And also use 12 hour format here

Reply:
Does this format look good? : ` 20 January 2021 at 07:06:20 PM `

Reply:
quite good.